src/azure-cli/HISTORY.rst

```diff
@@ -56,6 +56,7 @@ Release History
 * Fix issue #11100: AttributeError for az webapp up when create service plan
 * az webapp up: Forcing the creation or deployment to a site for supported languages, no defaults used.
 * Add support for App Service Environment: az appservice ase show | list | list-addresses | list-plans | create | update | delete
+* Fix issue #11217: webapp: az webapp config ssl upload should support slot parameter
```

Comment:
20.77 is the released version so don't add this here. Create an AppService section at the top (make sure to keep it alphabetic) & move this entry there.

Reply:
Thanks! I've updated now